dracut/module-setup.sh

```diff
@@ -58,6 +58,7 @@ install() {
     inst "$moddir/fetch-kickstart-disk" "/sbin/fetch-kickstart-disk"
     inst "$moddir/fetch-updates-disk" "/sbin/fetch-updates-disk"
     inst "$moddir/parse-kickstart" "/sbin/parse-kickstart"
+    inst "$moddir/version.py" "/sbin/version.py"
```

Comment:
I am not sure if you need to do this. We don't run this code as a script. It needs to be installed as a Python library. See the end of this file.

Reply:
Let's see how you like the one-liner needed to get a suitable directory :D